src/transformers/trainer.py

```diff
@@ -2096,8 +2104,8 @@ def _load_best_model(self):
                         state_dict["_smp_is_partial"] = False
                         load_result = model.load_state_dict(state_dict, strict=True)
                 elif self.is_fsdp_enabled:
-                    self.accelerator.state.fsdp_plugin.load_model(
-                        self.accelerator, model, self.state.best_model_checkpoint
+                    load_fsdp_model(
+                        self.accelerator.state.fsdp_plugin, self.accelerator, model, self.state.best_model_checkpoint
```

Comment:
This will require folks use yet another higher version of Accelerate, else they'll get a `load_fsdp_model` not found if they pin Accelerate but not Transformers. When 0.21.0 (of accelerate) is out we should either pin to it as the new minimum, or have an explicit error/check in here so it's not a confusing error for folks due to this naming change. 

I know fsdp is considered experimental to a degree, so trusting best judgement on if action is needed/what action to take :)